cheroot/ssl/builtin.py

```diff
@@ -67,19 +70,51 @@ class BuiltinSSLAdapter(Adapter):
     ciphers = None
     """The ciphers list of SSL."""
 
+    # from mod_ssl/pkg.sslmod/ssl_engine_vars.c ssl_var_lookup_ssl_cert
     CERT_KEY_TO_ENV = {
-        'subject': 'SSL_CLIENT_S_DN',
-        'issuer': 'SSL_CLIENT_I_DN',
+        'version': 'M_VERSION',
+        'serialNumber': 'M_SERIAL',
+        'notBefore': 'V_START',
+        'notAfter': 'V_END',
+        'subject': 'S_DN',
+        'issuer': 'I_DN',
+        'subjectAltName': 'SAN',
+        # not parsed by the Python standard library
+        # - A_SIG
+        # - A_KEY
+        # not provided by mod_ssl
+        # - OCSP
+        # - caIssuers
+        # - crlDistributionPoints
     }
 
+    # from mod_ssl/pkg.sslmod/ssl_engine_vars.c ssl_var_lookup_ssl_cert_dn_rec
     CERT_KEY_TO_LDAP_CODE = {
         'countryName': 'C',
-        'stateOrProvinceName': 'ST',
+        'stateOrProvinceName': 'ST',  # or 'SP' (SSLeay)
```

Comment:
Does this comment imply that the current implementation is imperfect?
I'm not sure how helpful it is in this state. Mind putting more details here? Maybe a `# TODO: ` or a `# Note: ` with the explanation of consequences of this.

Reply:
added a `# NOTE:`